### DISCUSSION TITLE
Order favourites by distance

### DISCUSSION THREAD

**Discussion by ThomasMeister123** [osmandapp/OsmAnd#15192] [Q&A] [answered]
Hello. I have a couple of favorites for a trip ("POI-todo-list"). How is it possible to order these by distance to my actual position? E.g. to see, what I can see aa next (nearest favorite).

**Comment by Boothy99** [comment, marked as answer]
To order the list by distance (from the centre of the map bring displayed) you need to use the search bar in My Places / with My Favourites selected, using at least part of the title of the favourites folder. 
For example, I have folders named Cornwall Beaches and Cornwall Walks. To view both of these folders within the search, I'd have to type ... Cornwall. Obviously if I was looking for the beaches only, I'd type ... Beaches. So if you wish to search both folders, the "Cornwall" has to be part of both folders names. The search results will be displayed in order of distance.
Hope that makes sense and helps?

**Reply by Boothy99** [reply]
![Screenshot_20220831-083854_OsmAnd+~2](https://user-images.githubusercontent.com/13983995/187621284-46a40de3-e7aa-4545-a200-46b863625344.jpg)
![Screenshot_20220831-083816_OsmAnd+~2](https://user-images.githubusercontent.com/13983995/187621288-b7496bbc-3e2e-4a02-85cc-3d5325afb871.jpg)

**Reply by Boothy99** [reply]
As you see from the second image, ... any category (favourites folder) with the word "Cornwall" has shown up in the list (canoeing, beaches, walks, pubs) ... in order of distance from where I'd zoomed the map to (Porthtowan in this case).

**Reply by ThomasMeister123** [reply]
OK, thank you. That works for me
